### DISCUSSION TITLE
Coffee Lake not supported?

### DISCUSSION THREAD

**Discussion by arskama** [opcm/pcm#295] [Q&A] [answered]
Hi,

I wanted to try pcm power on my coffee lake (i9-8950HK) and noticed surprisingly that it is not supported.
Is there any plan to support it?

**Comment by opcm** [comment]
the answer is here: https://github.com/opcm/pcm/blob/master/FAQ.md#q5

**Comment by opcm** [comment, marked as answer]
CPU energy metric is supported by the main pcm utility on Coffelake

**Reply by arskama** [reply]
Thanks for the answer!
